starboard/file.h

```diff
@@ -261,6 +261,9 @@ SB_EXPORT bool SbFileCanOpen(const char* path, int flags);
 // |mode|: The mode string to be converted into flags.
 SB_EXPORT int SbFileModeStringToFlags(const char* mode);
 
+// Moves file or directory.
+SB_EXPORT bool SbFileRename(const char* from_path, const char* to_path);
```

Comment:
We are not adding new SB file APIs in Starboard 16.
Also because of the difficulties in emulating `rename` semantics on some 1P platforms didn't add `SbFileRename`.

I guess we can discuss whether `rename` could be one of the new POSIX APIs in SB16.